hw2/04-background.js

```diff
@@ -1 +1,41 @@
 // Add your code here
+let input = document.getElementById("userinput");
+const background = document.querySelector("body");
+let button = document.getElementById("button");
+button.addEventListener("click", changColor);
+
+let interval;
+window.addEventListener("load", () => {
+  interval = setInterval(function () {
+    background.style.backgroundColor = color();
+  }, 3000);
+  button.value = "Stop";
+  button.style.background = "red";
+});
+
+function changColor() {
```

Comment:
I think the AirBnb Javascript style guide wants us to use named function expressions like so:

const name = function functionName() {

};

Otherwise this code looks good!

Reply:
Yes, I noticed it while I was reviewing the AirBnB reference, but I forgot to go back and change it. thanks.